### PR TITLE
Add frontmatter variable to hide edit url

### PR DESCRIPTION
**Pull request by SferaDev**
Closes https://github.com/primer/doctocat/issues/250

### Shows edit URL:

```
---
title: Using images
---
```

```
---
title: Using images
hideEditLink: false
---
```

### Hides edit URL:

```
---
title: Using images
hideEditLink: true
---
```

### CODE REVIEW

File: docs/content/index.mdx [hunk 1]
```diff
@@ -1,5 +1,6 @@
 ---
 title: Doctocat
+editable: false
```
Comment (SferaDev) [inline docs/content/index.mdx:3]: Not sure which pages should not be editable, feel free to revert this change

**Review by colebemis — COMMENTED** [review]
Hi, @SferaDev! Thanks for your patience ❤️ 

This is great! How do you feel about calling the frontmatter variable `hideEditLink` instead of `editable`? I think `hideEditLink` is a little more clear.

**Comment by SferaDev** [comment]
> How do you feel about calling the frontmatter variable `hideEditLink` instead of `editable`? I think `hideEditLink` is a little more clear.

Sure thing!

File: docs/content/usage/front-matter.mdx [hunk 2]
```diff
@@ -119,3 +119,13 @@ additionalContributors:
   - emplums
 ---
 ```
+
+## Hide edit this page link
```
Comment (colebemis) [inline docs/content/usage/front-matter.mdx:123]:
```suggestion
## Hide "Edit this page on GitHub" link
```

**Pull request closed without merging**